walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -65,14 +65,21 @@
       // if no openings for mand + opt, return mand only
       Collections.sort(mandBusy, TimeRange.ORDER_BY_START);
       return findOpenings(mandBusy, request.getDuration(), attendees);
-    } else {
-      // if openings exist for both or only contains opt attendees, return those
-      return mandOptOpenings;
     }
+    // if openings exist for both or only contains opt attendees, return those
+    return mandOptOpenings;
   }
 
   private static ArrayList<TimeRange> findOpenings(ArrayList<TimeRange> busy, long duration, Set<String> attendees) {
-    
+    /*
```

Comment:
Function documentation should be before the function header, in `/** ... */` JavaDoc style comments.